model-optimizer/extensions/middle/ONNXResize11ToInterpolate.py

```diff
@@ -34,9 +34,8 @@ def replace_resize(graph: Graph, resize: Node):
         log.warning('The input shape is not 4D or 5D for op with name {}'.format(resize_name))
         return
 
-    num_of_inputs = len([port for port in resize.in_ports().values() if not port.disconnected()])
-    assert num_of_inputs in {3, 4}, \
-        "Number of inputs of ONNXResize (with name {}) should be equal to 3 or 4".format(resize_name)
+    assert (resize.is_in_port_connected(0) and resize.is_in_port_connected(2)), \
+        "Data and scales inputs must be connected to Node {} with op {}.".format(resize.name, resize.op)
```

Comment:
use node.soft_get() for node.name and node.op

Reply:
`op` attribute always exist in the node, so no need to check it. But the `name` may be missing, so you must use `soft_get` for it.